.github/workflows/ci.yml

```diff
@@ -444,6 +445,7 @@ jobs:
             - { modules: plugin/trino-druid }
             - { modules: plugin/trino-iceberg }
             - { modules: plugin/trino-iceberg, profile: cloud-tests }
+            - { modules: plugin/trino-hudi }
```

Comment:
ideally these would be sorted, but they are not now..

Reply:
Added another commit to sort alphabetically (used IntelliJ's `Sort Lines`).